### PR TITLE
Added possibility to set a different mysql root user

### PR DESCRIPTION
**Pull request by jesusch**
On some plattforms (e.g. AWS RDS) the given root user
is named different (e.g. admin)

### CODE REVIEW

**Comment by jjethwa** [comment]
Hi @jesusch 

I'm away from my computer for 2 weeks.  I'll try to see if I can properly review your PR now otherwise it might have to wait.  Really appreciate it :smile:

File: Dockerfile [hunk 1]
```diff
@@ -13,65 +13,66 @@ ENV APACHE2_HTTP=REDIRECT \
...
-      snmp \
-      msmtp \
-      sudo \
-      supervisor \
-      unzip \
-      wget \
- && apt-get --purge remove exim4 exim4-base exim4-config exim4-daemon-light \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
+    && apt-get update \
```
Comment (bebehei) [inline Dockerfile]: Whats the purpose of this whitespace change?

**Comment by jesusch** [comment]
I guess this was my IDE

Benedikt Heine <notifications@github.com> schrieb am So., 27. Okt. 2019,
12:11:

**Comment by jjethwa** [comment]
Thanks @jesusch 
Really appreciate your work and PR.  Thanks for reviewing, @bebehei 😄

**Comment by bebehei** [comment]
As this is the most thread with the most recent activity: Sorry, but I'm out and will withdraw my elevated permissions and leave this repo. Since I'm not using icinga2 actively anymore, this might be the best solution. See you!

**Comment by jjethwa** [comment]
Thanks @bebehei  It was great working with you and I really appreciate everything you contributed.  Hope to meet in another project one day 😃

**Pull request merged**